### ISSUE TITLE
macOS 10.12 'could not access accessibility features' abort

### ISSUE THREAD

**Issue by OJFord** [koekeishiya/chunkwm#60]
I've built fine on macOS 10.12.4, but I can't run it:

> $ ./bin/chunkwm
> chunkwm: could not access accessibility features! abort..

It didn't ask me to allow it access, nor does it show in the list in Security & Privacy > Privacy > Accessibility; nor can I add it manually - it seems binaries cannot be added, only `.apps`.

Have I missed something? Cheers.

**Comment by OJFord** [comment]
Ah, fixed it. Restarting Finder.app made it start asking for permission, but then it still wouldn't pop up in system preferences from Alacritty.app (terminal emulator).

Switched to launching from Terminal.app and it worked fine :+1:.
